openedx/features/enterprise_support/api.py

```diff
@@ -458,7 +457,7 @@ def get_dashboard_consent_notification(request, user, course_enrollments):
                 enrollment = course_enrollment
                 break
 
-        client = ConsentApiClient()
+        client = ConsentApiClient(user=request.user)
```

Comment:
@haikuginger Do we need to use the service user when making this request to the consent_required API endpoint or are we good with the request user?

Reply:
Doesn't matter. Request user should work.